Tests/santa-driver/main.mm

```diff
@@ -0,0 +1,51 @@
+#import "SNTDriverManager.h"
```

Comment:
Can you add a comment at the top of this file describing what it's testing. I'm guessing it's a test that attempts to connect to the driver from multiple threads simultaneously doesn't cause crashes.

Reply:
You are right! I have added a comment at the top of the file